test/OpenTelemetry.Extensions.AWS.Tests/OpenTelemetry.Extensions.AWS.Tests.csproj

```
@@ -9,7 +9,7 @@
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Extensions.AWS\OpenTelemetry.Extensions.AWS.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != '$(NetFrameworkMinimumSupportedVersion)'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">
```

nit:

```suggestion
  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Reads a little nicer IMO 🤷